### PR TITLE
7095 start ws for all users

### PR DESCRIPTION
**Pull request by rosiehsmith**
- Start a websocket connection for all users when logging in
- Fix all the integration tests this broke
- Cleanup unused code
- add `maintenance_notify_west_object` so that websockets carry over to us-west if triggered from us-east

### CODE REVIEW

File: web-api/terraform/template/main-west.tf [hunk 1]
```diff
@@ -123,6 +134,20 @@ data "aws_s3_bucket_object" "websockets_green_west_object" {
   provider   = aws.us-west-1
 }
 
+data "aws_s3_bucket_object" "maintenance_notify_blue_west_object" {
+  depends_on = [null_resource.maintenance_notify_west_object]
+  bucket     = aws_s3_bucket.api_lambdas_bucket_west.id
+  key        = try("maintenance_notify_blue.js.zip", "maintenance_notify_green.js.zip")
```
Comment (matthopson) [inline web-api/terraform/template/main-west.tf]: Didn't this get changed?

Comment (matthopson) [reply web-api/terraform/template/main-west.tf]: This is what made it to mig: https://github.com/ustaxcourt/ef-cms/pull/1617/commits/9adc5ce786f1a3e09dd0885cf6678e6e4b17a10b

File: web-api/terraform/template/main-west.tf [hunk 2]
```diff
@@ -123,6 +134,20 @@ data "aws_s3_bucket_object" "websockets_green_west_object" {
...
+  provider   = aws.us-west-1
+}
+
+data "aws_s3_bucket_object" "maintenance_notify_green_west_object" {
+  depends_on = [null_resource.maintenance_notify_west_object]
+  bucket     = aws_s3_bucket.api_lambdas_bucket_west.id
+  key        = try("maintenance_notify_green.js.zip", "maintenance_notify_blue.js.zip")
```
Comment (matthopson) [inline web-api/terraform/template/main-west.tf]: Same here.

File: web-api/terraform/template/main-west.tf (hunk 1 above)
Comment (rosiehsmith) [reply web-api/terraform/template/main-west.tf]: 🤔 looks like some branch funkiness. thanks for pointing this out!

**Review by kkoskelin — APPROVED** [review]
Wow, nice work!

File: shared/src/notifications/retrySendNotificationToConnections.js [hunk 3]
```diff
@@ -12,9 +12,10 @@ const client = require('../persistence/dynamodbClientService');
 exports.retrySendNotificationToConnections = async ({
   applicationContext,
   connections,
+  deleteGoneConnections = true,
   messageStringified,
 }) => {
-  const maxRetries = 1;
+  const maxRetries = 3;
```
Comment (rachaelparris) [inline shared/src/notifications/retrySendNotificationToConnections.js]: Should we set this back to 1?

Comment (rosiehsmith) [reply shared/src/notifications/retrySendNotificationToConnections.js]: good idea!

**Review by codyseibert — APPROVED** [review]

**Pull request merged**